cmake/addons/depends/common/kodi-platform/kodi-platform.txt

```diff
@@ -1 +1 @@
-kodi-platform https://github.com/xbmc/kodi-platform e8574b883ffa2131f2eeb96ff3724d60b21130f7
+kodi-platform https://github.com/xbmc/kodi-platform master
```

Comment:
no, use the exact commit revision

Reply:
What, no release branch? Come on ...

This PR will stay as it is.

Reply:
then it won't get merged.